astronomer/providers/amazon/aws/triggers/emr.py

```diff
@@ -176,7 +148,7 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
                         "Try %s: Query is still in non-terminal state - %s", try_number, query_state
                     )
                     await asyncio.sleep(self.poll_interval)
-                if self.max_tries and try_number >= self.max_tries:
+                if self.max_retries and try_number >= self.max_retries:
```

Comment:
This implementation looks incorrect to me. Retries are meant for failures. But, looking at the logic here it means that every time a poll is made as part of the `while True` loop, when the query is in non-terminal state, the `try_number` is incremented. The variable is functioning more like `max_poll_attempts`

Reply:
Let me verify with sync version

Reply:
the behaviour looks correct to me maybe the variable name is a bit misleading here
https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/operators/emr.py#L209
https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/sensors/emr.py#L165

Reply:
okay. the docstring has the variable name `max_tries`. Can we keep it the same then? Retry would mean something failed and hence we're retrying. 
<img width="728" alt="Screenshot 2022-07-01 at 3 50 49 PM" src="https://user-images.githubusercontent.com/10206082/176876513-cc5f6c17-020a-46b9-8a53-1a74ae3d2c44.png">

Reply:
rename here b539cd168b71bda61d69c0542367d7d7e9d353b0